app/templates/gulp/gulp/tasks/scripts.js

```diff
@@ -11,7 +11,7 @@ var browserSync = require( 'browser-sync' );
 gulp.task( 'scripts', function() {
   return gulp.src( config.src )
     .pipe( $.sourcemaps.init() )
-    .pipe( $.concat( config.name ) )
+    .pipe( $.webpack() )
```

Comment:
Ok, webpack is building a string of random characters for the filename of the final JS file, so it's not loading on the demo page. Need to keep `config.name` in here, I'm guessing.